packages/lesswrong/components/recommendations/RecommendationsAndCurated.jsx

```diff
@@ -1,10 +1,10 @@
 import React, { PureComponent } from 'react';
 import { Components, registerComponent, withUpdate } from 'meteor/vulcan:core';
-import SettingsIcon from '@material-ui/icons/Settings';
+// import SettingsIcon from '@material-ui/icons/Settings';
```

Comment:
Deleted commented-out remnants

Reply:
not 100% sure what this means?

Reply:
(My intent is to leave in some commented out things that I planned to add back in relatively quickly. There's a comment later in the file to that effect. Not sure if you meant to say "delete" instead of "deleted")

Reply:
I meant to say delete (ie imperative), rather than deleted. Leaving commented-out code lying around is generally bad, even if there's a possibility of wanting it later. We can always find it in the commit history and bring it back that way.